### DISCUSSION TITLE
Bagaimana cara menambahkan data produk saat development?

### DISCUSSION THREAD

**Discussion by muchfahmiarif** [sensasi-apps/sensasi-pos#77] [General]
Om @sensasi-delight kalo diliat ini pake dexie, apakah fungsi CRUD pada dexie sudah berjalan normal untuk saat ini?
Dan apakah ada cara install dexie di linux environment saat develop?

**Comment by zainphp** [comment]
Fungsi CRUD untuk produk sudah berjalan normal pada #57, jika saat ini bermasalah kemungkinan karena perubahan struktur data tabel produk pada #74.

Untuk penggunaan Dexie pada Linux seharusnya tidak ada perbedaan pada sistem operasi lain, namun yang perlu dipastikan adalah semua pustaka telah terinstal dengan lengkap dan _browser_ sudah mendukung IndexedDB.
